msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/nativeauth/SignInEmailPasswordTest.kt

```diff
@@ -38,95 +41,70 @@ import org.junit.Ignore
 import org.junit.Test
 import org.mockito.Mockito
 import org.mockito.kotlin.spy
+import org.robolectric.RuntimeEnvironment.application
 
 class SignInEmailPasswordTest : NativeAuthPublicClientApplicationAbstractTest() {
 
-    override val defaultConfigType = ConfigType.SIGN_IN_PASSWORD
+    lateinit var application: INativeAuthPublicClientApplication
+    lateinit var config: NativeAuthTestConfig.Config
+
+    private val defaultConfigType = ConfigType.SIGN_IN_PASSWORD
+    private val defaultChallengeTypes = listOf("password", "oob")
 
     /**
      * Use valid email and password to get token.
      * (hero scenario 15, use case 1.2.1, Test case 37)
      */
     @Test
-    fun testSuccess() = runTest {
-        val username = config.email
-        val password = getSafePassword()
-        val result = application.signIn(username, password.toCharArray())
-        Assert.assertTrue(result is SignInResult.Complete)
+    fun testSuccess() {
+        config = getConfig(defaultConfigType)
+        application = setupPCA(config, defaultChallengeTypes)
+
+        runBlocking {
```

Comment:
Why do these tests not have retry operation? They are not supposed to be retried if they fail?

Reply:
Because they are email password sign in test case without otp requirement

Reply:
Ok, just in case, on IOS all tests are retried 5 times, so if there is a temporary 500 we don't mark the test as failed. Even the ones that don't retry the OTP

Reply:
Well confirmed. I'll stick to avoid retry logic here and try to keep it simple.